# assume that they goas like
# [up_p_r up_v_r mid_p_r mid_v_r feet_p_r feet_v_r {same for left} neck_p neck_v head_p head_v]
float64[16] vector_action
---
